packages/pigeon/lib/swift_generator.dart

```diff
@@ -671,17 +689,20 @@ private func nilOrValue<T>(_ value: Any?) -> T? {
 
     if (hasHostApi) {
       _writeWrapResult(indent);
-      _writeWrapError(indent);
+      _writeWrapError(generatorOptions, indent);
     }
     if (hasFlutterApi) {
-      _writeCreateConnectionError(indent);
+      _writeCreateConnectionError(generatorOptions, indent);
     }
+
+    _writePigeonError(generatorOptions, indent);
```

Comment:
One last nit and we can land it, can you move the pigeon error above the other methods here? above `if (hasHostApi) {'

Reply:
Done.